flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -166,7 +169,10 @@ public class Task
     /** ID which identifies the slot in which the task is supposed to run. */
     private final AllocationID allocationId;
 
-    /** TaskInfo object for this task. */
+    /** The meta information of current job. */
+    private final JobInfo jobInfo;
```

Comment:
Nit: This seems in the wrong commit.

Reply:
These code changes are prerequisite for the FLIP-380. I've moved these code changes to the commit `[FLINK-33905][core] Add getJobInfo() method to Environment`.